app/Http/Middleware/CheckFeature.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Ushahidi\App\Http\Middleware;
+
+use Ushahidi\App\Facades\Features;
+use Closure;
+
+class CheckFeature
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  mixed  ...$features
+     * @return mixed
+     */
+    public function handle($request, Closure $next, ...$features)
+    {
+        foreach ($features as $feature) {
+            if (!Features::isEnabled($feature)) {
+                abort(403, "Forbidden.");
```

Comment:
Robbie asked for a 404 when a feature is not enabled. But I'm not sure if we want to do it for everything or just the hdx feature. 

cc @rjmackay can you check this?

Reply:
Cool we can make it 404 instead with no issue, though it would be nice if this wasn't English it looks weird having a string attached to it. Surely, it can just be 404 and the browser figures it out